backend/src/main/java/com/now/naaga/common/exception/ControllerExceptionHandler.java

```diff
@@ -20,46 +19,41 @@ public class ControllerExceptionHandler {
     public ResponseEntity<ExceptionResponse> handleBaseException(final BaseException e) {
         final BaseExceptionType baseExceptionType = e.exceptionType();
         final ExceptionResponse exceptionResponse = new ExceptionResponse(baseExceptionType.errorCode(), baseExceptionType.errorMessage());
-        log.warn("error = {}", exceptionResponse, e);
+
+        log.info("error = {}", exceptionResponse);
+
         return ResponseEntity.status(baseExceptionType.httpStatus()).body(exceptionResponse);
     }
 
     @ExceptionHandler({
             HttpMessageNotReadableException.class,
             MethodArgumentNotValidException.class,
-            HttpMediaTypeNotSupportedException.class
-    })
+            HttpMediaTypeNotSupportedException.class})
     public ResponseEntity<ExceptionResponse> handleTypeMismatchException(final Exception e) {
         final CommonExceptionType commonExceptionType = CommonExceptionType.INVALID_REQUEST_BODY;
         final ExceptionResponse exceptionResponse = new ExceptionResponse(commonExceptionType.errorCode(), commonExceptionType.errorMessage());
-        log.warn("error = {}", exceptionResponse, e);
+
+        log.warn("error = {}", exceptionResponse);
```

Comment:
P4:

이부분을 따로 warn레벨로 설정하신 이유가 있으실까요?
이것또한 사용자의 요청이 잘못돼서 바인딩이 실패한 에러들이 대부분인데요.

인포로 내리고 어떤 예왼지 스택트레이스를 보여주는 것은 어떠신가요??

강한 주장은 아닙니다. 충분히 응답만 보고도 어떤 예왼지 알 수 있을 거에요.
```suggestion
        log.info(exceptionResponse, e);
```

Reply:
루카 말에 동의합니다
log.info(exceptionResponse, e);
가 좋은 것 같습니다

Reply:
저부분도 우리가 잡은 에러이군요 warn이 아니라 info 레벨이 맞는것같네요! 이부분으 수정 후 머지하겠습니다!

Reply:
log.info(exceptionResponse, e); 이렇게는 동작하지 않습니다 string, 그리고 그안에 값을 담는 형태여야하는데 "error = {}", exceptionResponse 이 형태는 마음에 안드시는것인지..!?